cascade-prelude/src/Cascade/Data/Chronos/Future.hs

```diff
@@ -0,0 +1,52 @@
+module Cascade.Data.Chronos.Future
+  ( Future
+  , ValidationError(..)
+  , ValidationErrors
+  , pattern Future
+  , un
+  , mk
+  , IsFuture
```

Comment:
Is this export obligatory? I guess it is due to the type of `mk`. Can you confirm it?

BTW, this module has a nice exposed API and is very neat in nature. Thank you.